packages/mobile/lib/auxiliary/drawer/actions/updateDrawerProps.action.ts

```diff
@@ -0,0 +1,15 @@
+import { DrawerId } from '../enums'
+import { drawersStore } from '../stores'
+import { DrawerProps } from '../types'
+
+export function updateDrawerProps(drawerId: DrawerId, props?: DrawerProps): void {
```

Comment:
This function should go inside the corresponding store file since it deals directly with data management. We can also call it `updateDrawerState` to be consistent with the other CRUD functions.